app/src/main/res/layout/fragment_subscriptions.xml

```diff
@@ -6,14 +6,14 @@
                 android:layout_height="match_parent">
 
     <androidx.appcompat.widget.Toolbar
-            android:layout_width="match_parent"
-            android:layout_height="wrap_content"
-            android:minHeight="?attr/actionBarSize"
-            android:theme="?attr/actionBarTheme"
-            android:layout_alignParentTop="true"
-            app:title="@string/subscriptions_label"
-            app:navigationIcon="?homeAsUpIndicator"
-            android:id="@+id/toolbar"/>
+                android:layout_width="match_parent"
```

Comment:
We are aiming for consistent use of 4 spaces. You replaced 8 spaces with 12 spaces here.